cmd/starter/c/starter.c

```diff
@@ -1398,10 +1398,10 @@ __attribute__((constructor)) static void init(void) {
      * the host libc library used by this starter binary.
      */
     if (getpwuid(0) == NULL) {
-        fatalf("Failed to retrieve root user information: %s\n", strerror(errno));
+        fatalf("Failed to retrieve root user information: %s\n", errno == 0 ? "Not Found" : strerror(errno));
```

Comment:
`Not found` => `root user not found`

Reply:
thanks! reflected here
https://github.com/apptainer/apptainer/pull/1483/commits/5e2eb22b69705515bd8a507b1598fa018d7015d4